test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -87,6 +91,31 @@ namespace iroha {
       std::string block_store_path = "/tmp/block_store";
     };
 
+    TEST_F(AmetsuchiTest, GetBlocksCompletedWhenCalled) {
+      // Commit block => get block => observable completed
+      auto storage =
+          StorageImpl::create(block_store_path, redishost_, redisport_, pgopt_);
```

Comment:
Because concrete `StorageImpl::getBlocks` implementation is tested, and `StorageImpl` depends on concrete classes, not interfaces. `StorageImpl` can be refactored in the future to use interfaces for block storage, index, and world state view.